doc/sphinx/source/recipes/recipe_smpi.rst

```diff
@@ -0,0 +1,93 @@
+
+Note: The SMPI diagnostic needs all indicated variables from all added models for exactly the same time period to be calculated correctly. If one model does not provide a specific variable, either that model cannot be added to the SMPI calculations, or the missing variable has to be removed from the diagnostics all together. 
+
+Available recipes and diagnostics
+-----------------------------------
+
+Recipes are stored in recipe/
+
+* recipe_reichlerkim08bams.yml
+
+Diagnostics are stored in diag_scripts/perfmetrics/
+
+* main.ncl: calculates and (optionally) plots annual/seasonal cycles, zonal means, lat-lon fields and time-lat-lon fields. The calculated fields can also be plotted as difference w.r.t. a given reference dataset. main.ncl also calculates RMSD, bias and taylor metrics. Input data have to be regridded to a common grid in the preprocessor. Each plot type is created by a separated routine, as detailed below.
+* cycle_zonal.ncl: calculates single model perfomance index (Reichler and Kim, 2008). It requires fields precalculated by main.ncl.
+* collect.ncl: collects the metrics previously calculated by cycle_latlon.ncl and passes them to the plotting functions.
+
+User settings
+-------------
+
+#. main.ncl
+
+   *Required settings for script*
+
+   * plot_type: cycle (time), zonal (plev, lat), latlon (lat, lon), cycle_latlon (time, lat, lon)
+   * time_avg: type of time average (opt argument of time_operations in diag_scripts/shared/statistics.ncl)
+   * region: selected region (see select_region in diag_scripts/shared/latlon.ncl)
+   * normalization: metric normalization (for RMSD and BIAS metrics only)
+   * calc_grading: calculates grading metrics (default: False; has to be set to "true")
+   * metric: chosen grading metric(s) (if calc_grading is True; has to be set to "SMPI")
+   * smpi_n_bootstrap: number of bootstrapping members used to determine uncertainties on model-reference differences
+   
```

Comment:
In the recipe there is an undocumented `range_option`, what is that about?

Reply:
That is a remnant from an older version of perfmetrics v2. I removed it from the recipe.